src/System.IO/src/System/IO/StreamReader.cs

```diff
@@ -155,6 +155,48 @@ public StreamReader(Stream stream, Encoding encoding, bool detectEncodingFromByt
             Init(stream, encoding, detectEncodingFromByteOrderMarks, bufferSize, leaveOpen);
         }
 
+        public StreamReader(string path)
+            : this(path, true)
+        {
+        }
+
+        public StreamReader(string path, bool detectEncodingFromByteOrderMarks)
+            : this(path, Encoding.UTF8, detectEncodingFromByteOrderMarks, DefaultBufferSize)
+        {
+        }
+
+        public StreamReader(string path, Encoding encoding)
+            : this(path, encoding, true, DefaultBufferSize)
+        {
+        }
+
+        public StreamReader(string path, Encoding encoding, bool detectEncodingFromByteOrderMarks)
+            : this(path, encoding, detectEncodingFromByteOrderMarks, DefaultBufferSize)
+        {
+        }
+
+        [System.Security.SecuritySafeCritical]
```

Comment:
I assumed so since they're included on the [mscorlib versions of these functions](https://github.com/dotnet/coreclr/blob/master/src/mscorlib/src/System/IO/StreamReader.cs#L191).